docs/README.md

```diff
@@ -6,25 +6,27 @@ Use this component to show badges based on your product specifications inside yo
+| `specificationName`           | `String`                                                  | Pass the name of the specification you want to target. If left empy, will target all of the group                                                                                                                                                                                       | `""`          |
+| `visibleWhen`                 | `String`                                                  | Pass this if you want the specification to be displayed only if it has this exact value. If left empty, the badge will be showed regardless of the specification value.                                                                                                                 | `""`          |
+| `displayValue`                | `SPECIFICATION_NAME` or `SPECIFICATION_VALUE` or `string` | Choose the value that will appear if the specification condition is met and the badge will be showed. Pass `SPECIFICATION_NAME` if you want to display the specification name. Pass `SPECIFICATION_VALUE` if you want to display its value. Pass any other custom string to display it. | `null`        |
+| `specificationValueSeparator` | `string`                                                  | Define the separator string to be rendered when a specification value has multiple values.                                                                                                                                                                                              | `/`           |
+| `specificationsOptions`       | `Array<Option>`                                           | Pass this if you want to control the conditions to show certain specifications. Each value of the array should be a valid object of the `Option` format.                                                                                                                                | `null`        |
+| `orientation`                 | `vertical` or `horizontal`                                | Determines if the group of specifications are showed horizontally or vertically                                                                                                                                                                                                         | `vertical`    |
+| `blockClass`                  | `string`                                                  | Allows to pass a custom name to be added to component css classes                                                                                                                                                                                                                       | `null`        |
```

Comment:
Can you order this alphabetically, please?

Reply:
for sure ma friend